spec/features/question_form_spec.rb

```diff
@@ -51,4 +51,22 @@
       end
     end
   end
+
+  scenario "audio upload works", js: true do
+    visit new_question_path(locale: "en", mode: "m", mission_name: mission.compact_name)
+    fill_in "Code", with: "AQuestion"
+    fill_in "Title", with: "Jay's"
+    select "Text", from: "Type"
+
+    attach_file("Audio Prompt", File.absolute_path("spec/fixtures/media/audio/powerup.mp3"))
+    click_on "Save"
+
+    visit edit_question_path(locale: "en", mode: "m",
+      mission_name: mission.compact_name, id: Question.last.id)
+
+    # on edit, paperclip does not show the audio file on the page even if it exists
+    # but we are certain that it gets saved
+    # perhaps this should be a model spec?
```

Comment:
For now I would say use the `show` page to verify the upload.

But you point out a usability problem -- if there is an existing value, it seems we should be showing it in the edit case. Can you please put that on your TODO list for the issue. We can address it later.